waku/v2/protocol/waku_archive/driver/postgres_driver.nim

```diff
@@ -0,0 +1,248 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  std/db_postgres,
+  std/sequtils,
+  std/parseutils,
+  std/strformat,
+  std/nre,
+  std/options,
+  stew/[results,byteutils],
+  std/strutils
+
+import
+  ../../../protocol/waku_message,
+  ../../../utils/time,
+  ../../../config,
+  ../common,
+  ../driver
+
+export postgres_driver
+
+type PostgresDriver* = ref object of ArchiveDriver
+  connection : DbConn
+  preparedInsert: SqlPrepared
+
+proc dropTableQuery(): string =
+  "DROP TABLE messages"
+
+proc createTableQuery(): string =
+  "CREATE TABLE IF NOT EXISTS messages (" &
+  " pubsubTopic VARCHAR NOT NULL," &
+  " contentTopic VARCHAR NOT NULL," &
+  " payload VARCHAR," &
```

Comment:
This should be a `bytea`, I will investigate a bit more on how to do that avoiding using `varchar`